src/components/Projects.tsx

```diff
@@ -1,7 +1,247 @@
 import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import {
+  space,
+  SpaceProps,
+  layout,
+  LayoutProps,
+  background,
+  BackgroundProps,
+  FlexboxProps,
+  flexbox,
+  PositionProps,
+  position,
+  ColorProps,
+  color,
+  TypographyProps,
+  typography,
+  border,
+  BorderProps,
+} from "styled-system";
+import stairs from "./assets/project-page/stairs.png";
+import shell from "./assets/project-page/shell.png";
+import eye from "./assets/project-page/eye.png";
+import statue from "./assets/project-page/statue.png";
+import dragon from "./assets/project-page/dragon.png";
+import knife from "./assets/project-page/knife.png";
+import mask from "./assets/project-page/mask.png";
+import spider from "./assets/project-page/spider.png";
+import magnify from "./assets/project-page/magnify.png";
+
+type BgProps = BackgroundProps &
+  ColorProps &
+  LayoutProps &
+  FlexboxProps &
+  TypographyProps &
+  SpaceProps;
+
+const Background = styled.div<BgProps>`
+  ${color};
+  ${layout};
+  ${flexbox};
+  ${background};
+  ${space};
+`;
+
+type HomeLinkProps = SpaceProps;
+const HomeLink = styled.a<HomeLinkProps>`
+  ${space};
+`;
+
+type HomeLogoProps = LayoutProps & TypographyProps & ColorProps;
+
+const HomeLogo = styled.p<HomeLogoProps>`
+  ${layout};
+  ${typography};
+  ${color};
+`;
+
+type BuyButtonProps = SpaceProps &
+  LayoutProps &
+  ColorProps &
+  TypographyProps &
+  BorderProps &
+  PositionProps &
+  SpaceProps &
+  FlexboxProps;
+
+const BuyButton = styled.button<BuyButtonProps>`
+  ${layout};
+  ${color};
+  ${typography};
+  ${border}
+  ${position};
+  ${space};
+  ${flexbox};
+`;
+
+type ProjectIconContainerProps = LayoutProps & PositionProps & FlexboxProps;
+
+const ProjectIconContainer = styled.div<ProjectIconContainerProps>`
+  ${layout};
+  ${position};
+  ${flexbox};
+`;
+
+type ProjectIconProps = LayoutProps & PositionProps & FlexboxProps & SpaceProps;
+
+const ProjectIcon = styled.a<ProjectIconProps>`
+  ${layout};
+  ${position};
+  ${flexbox};
+  ${space};
+  &:hover {
+    transition: transform 0.2s;
+    transform: scale(1.1);
+  }
+`;
 
 const Projects = () => {
-  return <div>Projects</div>;
+  const projectIconWidths = [50, 60, 80, 100, 120, 140];
+  return (
+    <Background
+      backgroundSize="cover"
+      backgroundImage="url('./assets/home-page/background.jpg')"
+      backgroundPosition="center"
+      backgroundRepeat="no-repeat"
+      bg="black"
+      height="100vh"
+      display="flex"
+      flexDirection="column"
+      alignItems="center"
+      mx="auto"
+    >
+      <HomeLink p={30}>
```

Comment:
HomeLink is an a tag
Link inside it is an a tag

![image](https://user-images.githubusercontent.com/11725595/86950208-2c1f1380-c150-11ea-91c3-1840fb891194.png)

Reply:
the same goes for all the icons below, they are `a` tags within `a` tagz

Reply:
Not sure I understand the error message here, are you saying that `a` tags within `a` should not be a thing?

Reply:
correct, you only need one `a` tag.

to apply styles to `react-router` `Link` we call on styled components:

```
const ProjectIconLink = styled(Link)`
  {styles go here as normal}
`
```

Reply:
A-ha! Got it